python/data/__init__.py

```diff
@@ -6,7 +6,7 @@
 from utils import logger
 
 from .dataloader import BatchSampler, DataLoader
-from .dataset.imagenet_dataset import ImageNetDataset
+from .dataset.imagenet_dataset import ImageNetDataset, Cityscapes
```

Comment:
Place CityScapes in seperate py file.
from .datase.cityscapes import CityScapesDataset